micrometer/src/main/java/feign/micrometer/FeignContext.java

```diff
@@ -29,6 +31,15 @@ public class FeignContext extends RequestReplySenderContext<Request, Response> {
   public FeignContext(Request request) {
     super(Request::header);
     setCarrier(request);
+    setRemoteServiceName(request.requestTemplate().feignTarget().name());
+    try {
+      URL url = new URL(request.requestTemplate().feignTarget().url());
+      if (StringUtils.isNotBlank(url.getProtocol()) && StringUtils.isNotBlank(url.getHost())) {
+        setRemoteServiceAddress(url.getProtocol() + "://" + url.getHost() + ":" + url.getPort());
+      }
+    } catch (Exception ignored) {
```

Comment:
what exception is been ignored here?

probably should re-throw it, as will be impossible for people to understand why something is not working if we are eating the exception